cli/kubectl-hyperkube

```diff
@@ -288,5 +299,42 @@ def _load_pem(pem):
         sys.exit(1)
 
 
+def _parse_kube_config(filename):
+    with open(filename) as data_file:
+        config = yaml.load(data_file, Loader=yaml.FullLoader)
+    return config
+
+
+def _merge(source_files, target_file):
+    parsed = [_parse_kube_config(file) for file in source_files]
+
+    # Now do an effective flatmap to get it all into a single structure
+    target_config = {
+        "apiVersion": "v1",
+        "kind": "Config",
+        "current-context": "",
+        "clusters": [],
+        "contexts": [],
+        "users": [],
+    }
+
+    for config in parsed:
+        for key in ['clusters', 'contexts', 'users']:
+            target_config[key].extend(config.get(key, []))
+        open(target_file, 'w').write(yaml.dump(target_config))
+
+
+def write_kubeconfig(fetched_kubeconfig, destination_file):
+    temp_kubeconfig_file = "temp-kubeconfig.yaml"
+    temp_file = open(temp_kubeconfig_file, 'w+')
+    yaml.dump(fetched_kubeconfig, temp_file, allow_unicode=True)
+    if os.path.exists(destination_file):
```

Comment:
maybe the following?

`_merge(destination_file, temp_kubeconfig_file, destination_file)` 

and then change _merge slightly
```
def _merge(target_file, *source_files):
    parsed = [_parse_kube_config(file) for file in source_files if os.path.exists(file)]
```